### ISSUE TITLE
Unify test design.

### ISSUE THREAD

**Issue by ElijahCaine** [osuosl/timesync-node#205]
There are a lot of different ways we write a lot of similar tests. We should consider changing them to be more unified to make auditing and revisions to the tests a bit easier.

@devs @Kennric Thoughts?

**Comment by LyonesGamer** [comment]
Agreed. It's something I've been meaning to look at designing for a little while.

**Comment by ElijahCaine** [comment]
To bring this discussion back
